cockroachdb/tests/test_unit.py

```diff
@@ -54,7 +55,9 @@ def test_metrics(aggregator, instance, dd_run_check, mock_http_response, file, m
     tags = ['cluster:cockroachdb-cluster', 'endpoint:http://localhost:8080/_status/vars', 'node:1', 'node_id:1']
 
     for metric in metrics:
-        aggregator.assert_metric('cockroachdb.{}'.format(metric), tags=tags)
+        aggregator.assert_metric('cockroachdb.{}'.format(metric))
+        for tag in tags:
+            aggregator.assert_metric_has_tag('cockroachdb.{}'.format(metric), tag)
```

Comment:
This is what made you add the `assert_metric_has_tags` method on the aggregator. Nice